build.gradle.kts

```diff
@@ -17,7 +17,7 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-thymeleaf")
   implementation("nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect:2.4.1")
 
-  testImplementation("net.javacrumbs.json-unit:json-unit-assertj:2.18.0")
-  testImplementation("org.testcontainers:localstack:1.13.0")
+  testImplementation("net.javacrumbs.json-unit:json-unit-assertj:2.18.1")
+  testImplementation("org.testcontainers:localstack:1.14.3")
```

Comment:
very brave move :-)

Reply:
didn't even notice, just did an upgrade of dependencies :shrug: